src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1354,25 +1352,29 @@ private void InitEditorFresh()
 
     private void InitEditorSaved()
     {
-        UpdateGUIAfterLoadingSpecies(editedSpecies);
+        UpdateGUIAfterLoadingSpecies(startingSpecies);
         OnLoadedEditorReady();
     }
 
     private void SetupEditedSpecies(MicrobeSpecies species)
     {
-        editedSpecies = species ?? throw new NullReferenceException("didn't find edited species");
+        startingSpecies = species ?? throw new NullReferenceException("didn't find edited species");
+        CurrentSpecies = (MicrobeSpecies)species.Clone();
```

Comment:
Why are you cloning the species if you anyway explicitly copy the membrane etc. properties a few lines later?